index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
```

Comment:
I would suggest you make use of [html semantic tags](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) and read up on [html best pratices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).